docs/index.asciidoc

```diff
@@ -424,6 +425,17 @@ When the `target` is set to a field reference, the `_source` of the hit is place
 This option can be useful to avoid populating unknown fields when a downstream schema such as ECS is enforced.
 It is also possible to target an entry in the event's metadata, which will be available during event processing but not exported to your outputs (e.g., `target \=> "[@metadata][_source]"`).
 
+
+[id="plugins-{type}s-{plugin}-retries"]
+===== `retries`
+
+* Value type is <<number,number>>
+* Default value is `0`
+
+The number of retries to run the query. If the query fails after all retries, it logs an error message.
+The default is 0, no retry. This value should be equal or greater than zero.
+
+
```

Comment:
@karenzone I have added `retries`option for the plugin. Please review the doc

Reply:
Nice, thank you!

Reply:
Should be in alphabetical order between `request_timeout` and `schedule`

Reply:
:facepalm: I renamed `tries` to `retries` but forgot to move it alphabetically